scripts/build-deb.sh

```diff
@@ -13,7 +13,7 @@ ensureAvailable lintian
 ensureAvailable rpmbuild
 
 PACKAGE_TMPDIR=tmp/debian_pkg
-VERSION=`dist/bin/yarn --version`
+VERSION=`./artifacts/yarn-legacy-* --version`
```

Comment:
Yeah we can. I'd leave this one though since this script doesn't rely on `dist` being there, but the other one does so I'll revert that one before landing.